pkg-svcs/backend/.env.registry

```diff
@@ -6,38 +6,33 @@ API_PORT=3000
 
 CORS_ORIGIN=true
 
-PG_HOST=postgresql
-PG_PORT=5432
-PG_DB=tododb
-PG_USER=otlpuser
-PG_PASS=otlppassword
-
+# The DB info stored in the secret $SECRETS_PG_INFO should come in JSON form, like this:
+# {"database": "database", "username": "username", "password": "password", "host": "host", "port": port}
 SECRETS_STRATEGY=aws
-SECRETS_PG_PASS=DB_PASSWORD
 
 OTLP_DEBUG=true
 OTLP_SERVICE_NAME=oltp-backend
 OTLP_ENVIRONMENT=development
 OLTP_ENABLE_CONSOLE_LOG=true
-OLTP_COLLECTOR_URL=http://dev-otlp-lb-571900603.us-east-1.elb.amazonaws.com:8083/v1/traces
+OLTP_COLLECTOR_URL=http://dev-otlp-lb-1255345766.us-east-1.elb.amazonaws.com:8083/v1/traces
```

Comment:
we need to make this somehow dynamic or provide enough instructions to reflect its update. Also what when CI will be enabled?